datalad/support/annexrepo.py

```diff
@@ -130,6 +130,13 @@ def __init__(self, path, url=None, runner=None,
         batch_size: int, optional
           if specified and >0, instructs annex to batch this many commands before
           annex adds acts on git repository (e.g. adds them them to index for addurl).
+
```

Comment:
Now it is set in stone: https://github.com/datalad/datalad/blob/master/CONTRIBUTING.md#docstrings 
As for different styles in the files -- I think it is primarily someones were lazy whenever using PyCharm which didn't yet support NumPy style docstrings.  Now it does ;)